article-analyzer/src/main/java/co/inventorsoft/academy/spring/config/Configuration.java

```diff
@@ -0,0 +1,33 @@
+package co.inventorsoft.academy.spring.config;
+
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import java.util.Set;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+
+/**
+ * Configuration file.
+ */
```

Comment:
the name of the class tells what it is, remove this doc

Reply:
If changed into more meaningful sentence, can it be left?

Reply:
if so, sure